modules/persistenceService/src/main/resources/db/migration/V2__ALTER_USER_TABLE_DROP_ADD_NEW_COLUMN.sql

```diff
@@ -15,6 +15,6 @@ CREATE TABLE subscriptions(
     id uuid NOT NULL,
     repository_id uuid NOT NULL,
     PRIMARY KEY(id, repository_id),
-    FOREIGN KEY (id) REFERENCES userdb(id) ON UPDATE CASCADE,
-    FOREIGN KEY (repository_id) REFERENCES repositories(repository_id) ON UPDATE CASCADE
+    FOREIGN KEY(id) REFERENCES userdb(id) ON UPDATE CASCADE,
+    FOREIGN KEY(repository_id) REFERENCES repositories(repository_id) ON UPDATE CASCADE
```

Comment:
Comment: Flyway generates a checksum of the applied migrations and then checks the value when applying the migrations. This means that we shouldn't modify any existing migration that has been merged.

Reply:
ohh so even a space in the file will cause an issue. I didn't know that before. 
Thank you Fede